src/Escaper/Escaper.h

```diff
@@ -0,0 +1,107 @@
+/**@file
+ * This file is part of the TASTE Runtime Common.
+ *
+ * @copyright 2021 N7 Space Sp. z o.o.
+ *
+ * TASTE Runtime Common was developed under a programme of,
+ * and funded by, the European Space Agency (the "ESA").
+ *
+ * Licensed under the ESA Public License (ESA-PL) Permissive,
+ * Version 2.3 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     https://essr.esa.int/license/list
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+#ifndef ESCAPER_H
+#define ESCAPER_H
+
+/**
+ * @file    Escaper.h
+ * @brief   Main header file for escaping mode
+ *
+ */
+
+#include <stdint.h>
+#include <stdbool.h>
+#include <stddef.h>
+
+#include <Broker.h>
```

Comment:
Ok, Broker is still required here for `BROKER_BUFFER_SIZE`. Shall we do something with it?

Reply:
Finished